### ISSUE TITLE
All users are sysops (and I can't un-sysop them)

### ISSUE THREAD

**Issue by kstrauser** [NuSkooler/enigma-bbs#253]
For :bug: bug reports, please fill out the information below plus any additional relevant information. If this is a feature request, feel free to clear the form.

**Short problem description**

I've freshly installed 0.0.10-alpha. I noticed today that _all_ of my new users have group "sysops".

As a fun side note, when I run `./oputil.js user group my_user -sysops` I get an error: "Object does not exist: No matching username". This works for all values of `-groupname`:

```shell
$ ./oputil.js user group my_user "+hithere2"
User groups modified
$ ./oputil.js user group my_user "-hithere2"
Object does not exist: No matching username
```

As an experiment, I tried using the `--` argument to tell Node to stop parsing the rest of the command line as options. This worked:

```shell
$ ./oputil.js user group my_user -- "-hithere2"
User groups modified
```

I'm including this information so that anyone searching for that error message can find the workaround.

**Environment**
- [X] I am using Node.js v10.x LTS or higher
- [X] `npm install` or `yarn` reports success
- Actual Node.js version (`node --version`): v10.18.1
- Operating system (`uname -a` on *nix systems): Linux myhost 5.0.0-31-generic #33~18.04.1-Ubuntu SMP Tue Oct 1 10:20:39 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
- Revision (`git rev-parse --short HEAD`): 2e52e90

**Expected behavior**

Only the first user is automatically made sysop.

**Actual behavior**

We're all sysops up in here.

**Steps to reproduce**

Create an account. Voila! You're a sysop!

**Comment by NuSkooler** [comment]
This isn't something that others have reported or that I've seen anywhere else. New users are assigned to the groups defined in your `config.hjson` `defaultGroups`. What does your config look like? By default (e.g. if you don't override the setting) that is just [users](https://github.com/NuSkooler/enigma-bbs/blob/0.0.10-alpha/core/config.js#L214)

**Comment by arcdigital** [comment]
I'm seeing this issue as well, my config file doesn't have anything explicit set for `defaultGroups`.

**Comment by NuSkooler** [comment]
I'll see if I can repo this today.

**Comment by NuSkooler** [comment]
OK found the bug and pushed a fix to `master` / `0.0.10-alpha` and `0.0.11-beta`: The default groups were getting modified when creating the initial sysop user. What that means is this would only happen if you launch a fresh copy -> create a new user (which which will be user ID = 1 = sysop) then create additional users. If you restart the system, no additional users would fall in the `sysops` group. Either way, it should be good to go now.

As for the `oputil.js` weirdness, it can depend on your shell as well. Feel free to file a separate ticket for that. Maybe we can use another syntax than "+" and "-".

**Comment by kstrauser** [comment]
Awesome. Thanks, @NuSkooler!
